### PR TITLE
feat(KtFieldToggle): change color of checkbox bg

### PR DESCRIPTION
**Pull request by carsoli**
change color of checkbox on hover

remove dead focus-within class

remove argument of toggle-colors mixin related to stroke color of check

introduce argument for bg color to mixin

### CODE REVIEW

**Comment by carsoli** [comment]
https://www.figma.com/file/0yFVivSWXgFf2ddEF92zkf/Kotti-Design-System?node-id=3328%3A7269&t=M4CdKW7ThCyBqHgL-0

see design here for changes to check and hover colors

(no decision on indeterminate state changes is done and so it's not part of the PR)

**Comment by carsoli** [comment]
Alberto prefers that we remove the css shadow all together from the checbox styling

File: packages/kotti-ui/source/kotti-field-toggle/components/ToggleBox.vue [hunk 1]
```diff
@@ -30,15 +30,19 @@ export default defineComponent({
 <style lang="scss">
 @import '../../kotti-field/mixins';
 
-@mixin toggle-colors($check-stroke-color, $border-color, $shadow-main) {
+@mixin toggle-colors($border-color, $shadow-main, $background-color) {
```
Comment (santiagoballadares) [inline packages/kotti-ui/source/kotti-field-toggle/components/ToggleBox.vue:33]: Not sure if we keep arguments in alphabetical order, if so then:
```suggestion
@mixin toggle-colors($background-color, $border-color, $shadow-main) {
```

Comment (carsoli) [reply packages/kotti-ui/source/kotti-field-toggle/components/ToggleBox.vue:33]: not really, i think those were sorted by importance or so

**Comment by santiagoballadares** [comment]
FT done. LGTM

**Review by Isokaeder — APPROVED** [review]

**Pull request merged**